### DISCUSSION TITLE
Change arguments in IFunctionInvocationFilter?

### DISCUSSION THREAD

**Discussion by jorisdg** [microsoft/semantic-kernel#9631] [Q&A] [answered]
You can return a new function result from the filter, but is it possible to change arguments before making the next(context) call?

**Comment by dmytrostruk** [comment, marked as answer]
@jorisdg Thanks for your question! 

Yes, it's possible to change arguments before making the `next(context)` call, here is an example:
https://github.com/microsoft/semantic-kernel/blob/7d9f834130042f4eb50946dbc94667270157b22a/dotnet/samples/Concepts/Filtering/FunctionInvocationFiltering.cs#L179-L184

**Reply by jorisdg** [reply]
Oh I totally missed that. Thank you, this was exactly what I needed!!
